sdcm/utils/distro.py

```diff
@@ -43,6 +43,7 @@ class Distro(enum.Enum):
     UBUNTU18 = ("ubuntu", "18.04")
     UBUNTU20 = ("ubuntu", "20.04")
     UBUNTU21 = ("ubuntu", "21.04")
+    UBUNTU21_10 = ("ubuntu", "21.10")
```

Comment:
21.10 is not LTS, and we had no non-LTS before. So, why is it added?

Reply:
since locally i'm running with ubuntu 21.10, some
of the docker/k8s local test won't run with this.